### PR TITLE
refactored to use github builds

### PR DESCRIPTION
**Pull request by nieuwsma**
### Summary and Scope

refactored to use github action builds
update alpine version

### Issues and Related PRs

* Resolves CASMHMS-5361

### Testing

Tested on:

currently depending on building as my only measure of testing.  

Probably should execute on a live system.

### Risks and Mitigations

Updates the alpine version to 3.15 which is recommended for security

### CODE REVIEW

**Comment by rsjostrand-hpe** [comment]
I tried running the docker image built from this PR and bash is missing:
```
[~]$ docker run --rm -it artifactory.algol60.net/csm-docker/unstable/hms-pytest:2.0.0-20220204140536.f5e7fb6
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "/bin/bash": stat /bin/bash: no such file or directory: unknown.
```

@schooler-hpe is the hms-pytest image expected to have bash? I see `/bin/bash` is specified for the CMD directive, but is not installed in the Dockerfile. https://github.com/Cray-HPE/hms-pytest/blob/master/Dockerfile#L41

**Comment by nieuwsma** [comment]
> 

makes me wonder how it works today.

**Comment by rsjostrand-hpe** [comment]
The command `pytest` gets invoked here: https://github.com/Cray-HPE/hms-test/blob/master/cmd/hms-pytest#L51 so maybe that is why it works.

**Comment by schooler-hpe** [comment]
BASH isn't needed in the image by the tests, but that does look incorrect as it is. It would be good to fix that error.

I ran the CT tests on Mug using the image built from this PR. Everything passed and looked good.

File: .license_check.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+add_exclude:
+  - ".license_check.yaml"
+  - ".github/*"
+  - "Makefile"
```
Comment (shunr-hpe) [inline .license_check.yaml:4]: Does the Makefile need to be ignored?

**Review by shunr-hpe — APPROVED** [review]

File: .license_check.yaml (hunk 1 above)
Comment (nieuwsma) [reply .license_check.yaml:4]: Need to... not strictly. but im getting awful annoyed at the license checker.  And it is JUST configuration.

**Review by schooler-hpe — APPROVED** [review]

**Pull request merged**